src/Sulu/Bundle/PreviewBundle/Resources/config/config_preview.yml

```diff
@@ -1,5 +1,15 @@
+parameters:
+    sulu.preview.voter_class: 'Sulu\Bundle\PreviewBundle\Security\PreviewVoter'
```

Comment:
we does not use this parameter anymore (symfony has kicked out this parameters from best-practice)